### DISCUSSION TITLE
Welcome to sakai-trinity Discussions!

### DISCUSSION THREAD

**Discussion by profmikegreene** [profmikegreene/sakai-trinity#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by profmikegreene** [comment]
Mostly just wanted to see what these looked like as I haven't tried them recently.

**Reply by curiousmustard** [reply]
Reply!
